results.tex

```diff
@@ -36,7 +37,7 @@ \subsection{Nuclear Material Inventory}
 ton of \gls{UOX} fuel, about nine times of tails is produced. 
 Spent fuel is discharged from reactors every refueling period.
 The entire core is discharged when the reactor decommissions.
-A total of about $1,000,000 MTHM$ of tails and $100,000 MTHM$ of
+A total of about $1,000,000$MTHM of tails and $100,000$MTHM of
 \gls{UNF} accumulate in 2050.
```

Comment:
It didn't all accumulate in one year. Instead, say: "A total of about $1,000,000$MTHM of tails and $100,000$MTHM of \gls{UNF} have accumulated by 2050"